tests/test_example_dags.py

```diff
@@ -46,3 +46,23 @@ def test_example_dag(session, dag_id):
     if dag is None:
         raise NameError(f"The DAG with dag_id: {dag_id} was not found")
     test_utils.run_dag(dag, account_for_cleanup_failure=True)
+
+
+@pytest.mark.skip(
+    airflow.__version__ >= "2.3.0", reason="Require Airflow version > 2.3.0"
+)
```

Comment:
```suggestion
@pytest.mark.skipif(
    airflow.__version__ < "2.3.0", reason="Require Airflow version > 2.3.0"
)
```

Reply:
https://docs.pytest.org/en/7.1.x/how-to/skipping.html#id1

Reply:
i.e. you want to skip if the version < 2.3

Reply:
yes :)